content/sensu-go/6.2/operations/deploy-sensu/secure-sensu.md

```diff
@@ -83,11 +83,11 @@ For communication to continue, you must complete the configuration in this secti
 
 Configure the following backend secure sockets layer (SSL) attributes in `/etc/sensu/backend.yml`:
 
-1. Replace the placeholders with the paths to your certificate and key files for the `cert-file`, `key-file`, and `trusted-ca-file` parameters:
+1. Replace the placeholders with the paths to your CA root, agent certificate, and agent key files for the `trusted-ca-file`, `cert-file`, and `key-file` parameters:
 {{< code yml >}}
-cert-file: "/path/to/ssl/cert.pem"
-key-file: "/path/to/ssl/key.pem"
-trusted-ca-file: "/path/to/trusted-certificate-authorities.pem"
+trusted-ca-file: "/etc/sensu/tls/ca.pem"
+cert-file: "/etc/sensu/tls/agent.pem"
+key-file: "/etc/sensu/tls/agent-key.pem"
```

Comment:
If these two parameters are supposed to be set in backend.yml, then this seems to contradict the descriptions we are using for these parameters in the backend reference. See https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/backend/#security-configuration-flags:

- For `cert-file`, the backend reference says "Path to the primary backend certificate file"
- For `key-file`, the backend reference says "Path to the primary backend key file."

Should these two parameters use the backend-1.pem and backend-1-key.pem values instead?

Reply:
@hillaryfraley yep, they should. good catch